### ISSUE TITLE
Only indicate availability of DC voltage for facilities

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/peeringdb#1341]
**Is your feature request related to a problem? Please describe.**
Inspired by the discussion in #1332, I've rethought the whole voltages. IMO, it would only make sense to indicate whether a facility offers DC or not. The AC voltage is defined by the location (i.e. country) of the facility.

**What is the impact?**
Make information more useful

**Describe the solution you'd like**
Remove any AC voltage and only have tickbox for 48VDC

**Do you think this feature will require a formal design?**
no

**Comment by martinhannigan** [comment]
I agree we should remove common voltages like 120,208, (removing 240) and
230. We already have a check box for neg 48. No AC voltage indication would
be a step backwards, provide less site information for interconnection
location choices, and potentially reduce choice (competition) in a
location. I agree less is more in this case and it is worth getting right.
Removing AC altogether seems like overkill, although there was already
discussion in other threads about doing just that.

-1

**Comment by arnoldnipper** [comment]
I'm fine of leaving AC in, generally. May I have a +1, @martinhannigan, please :)

**Comment by martinhannigan** [comment]
Of course, but what am I supposed to be agreeing to? We already have 48VDC.

**Comment by arnoldnipper** [comment]
Let me rephrase. Let's only have

- 48VDC
- AC

**Comment by martinhannigan** [comment]
Thanks Arnold. This doesn’t make a lot of sense.

-1

**Comment by arnoldnipper** [comment]
> Thanks Arnold. This doesn’t make a lot of sense.

What is your constructive proposal? You said:

>I agree we should remove common voltages like 120,208, (removing 240) and 230.

What's then left the facility admins chose from?

**Comment by martinhannigan** [comment]
If you look at the current list, it consists of

48 VDC
120 VAC (extremely common)
208 VAC (extremely common)
240 VAC (extremely common) [ Removing ]
400 VAC (less common, distribution) [ implementing ]
480 VAC (less common, distribution)

The next power list restructure could look like:

48 VDC (medium common, inverters, DC plants, etc.)
400 VAC (less common)
480 VAC (less common)

When we decided to put this data in we also decided we would go slow since this is a complex topic and under or overproviding data can lead to misuse or confusion. This cycle makes sense to me.

Thanks

-M<



Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by arnoldnipper** [comment]
> The next power list restructure could look like: 48 VDC (medium common, inverters, DC plants, etc.) 400 VAC (less common) 480 VAC (less common)

As I learned yesterday, 380 VDC also is around.

**Comment by martinhannigan** [comment]
Its an outlier at the far edge of the curve -2 for buildings and its a
custom offering in DC’s. Like very common voltage, this is very uncommon
contrasted to -1 curve wise for the others.

I’d prefer to keep the issue as-is and reconsider 380VDC later.

Tks

-M<

**Comment by mcmanuss8** [comment]
+1

**Comment by martinhannigan** [comment]
+1

From: mcmanuss8 ***@***.***>
Date: Thursday, April 6, 2023 at 12:16 PM
To: peeringdb/peeringdb ***@***.***>
Cc: Martin Hannigan ***@***.***>, Mention ***@***.***>
Subject: Re: [peeringdb/peeringdb] Only indicate availability of DC voltage for facilities (Issue #1341)

+1

**Comment by grizz** [comment]
**Summary**
Change voltages choices to 

400 VAC
480 VAC
48 VDC

**Comment by martinhannigan** [comment]
+1

**Comment by ehohermuth** [comment]
We should either provide all diffrent currents https://en.wikipedia.org/wiki/Mains_electricity_by_country to be neutral or none.

**Comment by eprosenx** [comment]
I am torn on this topic.  As an engineer, I like precision and accurate data.  The trick is that if we want to fully model this, we can go A LONG WAYS down the rabbit hole.

In the US alone we have the following: (and likely more)
1. 120/240 split phase (extremely common in residential and small business)
2. 120/208v three phase, neutral grounded (extremely common)
3. 277/480 three phase, neutral grounded (extremely common)
4. 120/240v high leg delta (somewhat uncommon)
5. 240v corner grounded delta (very uncommon)

Even within the above, there are scenarios even in my employers facilities where we only have 120v inverters off our -48v plants, so we would need a selection for 120v separate from 208v or 240v.

In Europe some systems are 230v phase to phase, others are 230v phase to neutral.  Folks might want that modeled differently as fault current scenarios and ratings are different for wire.

On top of this, it might be super useful to know who provides UPS backed AC vs. just straight mains current.  What about straight mains current but that is generator protected with an ATS?  How about A/B redundant feeds (but then we could go rapidly down the path of what does A/B mean and there are many sub types...)

Many facilities may provide 120/208v UPS backed, but then have 277/480 available but not UPS backed (though they may have it both generator and non generator backed - see the Pittock Building or Westin Building).

FWIW, for the most part mainline modern IT equipment is mostly good for between 100v and 240v mains current so this is not as much of an issue.  Most facilities are capable of at least 200v service (Japan) which works with the "high power draw" power supplies that can't do the lower voltages.

With all of that being said, I would suggest simplifying this down to perhaps two boolean fields: UPS Backed AC available, Battery Backed -48v DC available.  Those are probably the most key fields to be filtering on, and then from there prospective tenants should probably have a conversation with the landlord about details.

PeeringDB does not have to have all the answers.  ;-)

**Comment by martinhannigan** [comment]
We dont need to know every voltage. We wanted to identify uniqueness. 480V
as a more unique service offering. 208v and 120v is not.

We also didn't attempt to define the world. If someone wants to open an
issue for a region and propose we add unique voltage attributes for site
selection thats great.

120v,208v,240v are common (and boring). We did go a very long way down the
rabbit hole already.

Hope that helps

-M<
